packages/sdk-browser/src/utils.ts

```diff
@@ -28,11 +28,30 @@ export const createEmbeddedAPIKey = async (
     aead: AeadId.Aes256Gcm,
   });
 
+  // Function to extract x and y components from a hex string public key
+  function getXYComponentsFromHexString(publicKeyHex: string) {
+    publicKeyHex = publicKeyHex.replace(/^0x/, '');
+    if (!publicKeyHex.startsWith('04')) {
+      throw new Error("Public key is not in uncompressed format");
+    }
+    const x = publicKeyHex.slice(2, 66);
+    const y = publicKeyHex.slice(66, 130);
+    return { x, y };
+  }
```

Comment:
This is reminiscent of `pointDecode`: https://github.com/tkhq/sdk/blob/08e2b8c30848498ed0850aa1e1ede67775b001e5/packages/api-key-stamper/src/tink/elliptic_curves.ts#L135-L159

Except that here we're decoding from an **uncompressed** public key (which has both x and y components). Might make sense to unify and support both uncompressed and compressed hex strings in `pointDecode`?`pointDecode` ensures the x and y are valid P256 point coordinates, which is nice. Not sure if we could provide the same guarantee with uncompressed pub keys?

Reply:
Thanks for the callout here -- I was just looking into uncompressed vs compressed. I'll look into these and consider adding uncompressed support to PointDecode

Reply:
Added uncompressed checking to point decode